au/code/au/power_aliases.hh

```diff
@@ -45,29 +46,39 @@ template <typename T>
 constexpr auto inverse(T x) -> decltype(pow<-1>(x)) {
     return pow<-1>(x);
 }
+template <typename T>
+using Inverse = decltype(inverse(std::declval<T>()));
 
 // Make "squared" an alias for "pow<2>" when the latter exists (for anything).
 template <typename T>
 constexpr auto squared(T x) -> decltype(pow<2>(x)) {
     return pow<2>(x);
 }
+template <typename T>
+using Squared = decltype(squared(std::declval<T>()));
 
 // Make "cubed" an alias for "pow<3>" when the latter exists (for anything).
 template <typename T>
 constexpr auto cubed(T x) -> decltype(pow<3>(x)) {
     return pow<3>(x);
 }
+template <typename T>
+using Cubed = decltype(cubed(std::declval<T>()));
 
 // Make "sqrt" an alias for "root<2>" when the latter exists (for anything).
 template <typename T>
 constexpr auto sqrt(T x) -> decltype(root<2>(x)) {
     return root<2>(x);
 }
+template <typename T>
+using Sqrt = decltype(sqrt(std::declval<T>()));
```

Comment:
Broader question: Is this the name we want to use for these helpers.  I know when I see `cbrt` I have to look up what that means.  Our API is pretty fluent, so `SquareRoot` and `CubeRoot` wouldn't really be out of place.

Reply:
Hmm... I think so?

The argument for the status quo is that `cbrt` should be consistent with `std::cbrt`, so we make `Cbrt` consistent with `cbrt`.  This makes it easier for people to "guess" the rest if they know any of them.

I do agree that for type names, `CubeRoot<Nano<Liters>>` would be nicer than `Cbrt<Nano<Liters>>` in a lot of ways, but for now at least I'm finding the consistency argument more persuasive.

Reply:
Fair enough.  The lower case matching up is obviously plays well with `std`.  And I don't think we should be polluting the namespace with a bunch of aliases.  Projects can always provide their own alias if desired.